.env.example

```
@@ -0,0 +1,16 @@
+# App Environment 
+APP_PRODUCTION=false 
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)